### PR TITLE
CI Pipeline: Add integration test for get step

### PR DESCRIPTION
**Pull request by iAmoric**
Note for other reviewers on why this is required:

before we counted on implicit get to do this test for us... and until https://github.com/Pix4D/concourse-pypi-resource/pull/91 it worked fine... what we called before pypi-acceptance-test was testing both put and get steps... after I disabled the implicit get, I basically removed half of the tests (without thinking)...

The approach proposed here adds back the get step tests but in a separate job to conform to OPA rules (no implicit get). This is also clearer and the same approach as in: https://builder.ci.pix4d.com/teams/developers/pipelines/s3-resource-simple-master

This clarity with separation test-put and test-get also bring us the confidence that none will just remove half of the acceptance tests in future by mistake again...

### CODE REVIEW

**Comment by Pix4D-Janus** [comment]
pipeline_setter_bot: your pipeline is set:
https://builder.ci.pix4d.com/teams/developers/pipelines/concourse-pypi-resource-pipeline-re-add-get-test

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml [hunk 1]
```diff
@@ -188,6 +188,23 @@ jobs:
     params:
       glob: 'dist/*'
 
+- name: concourse-pypi-resource-integration-test-get
```
Comment (aliculPix4D) [inline ci/pipelines/concourse-pypi-resource-pipeline-template.yml:191]: Note for other reviewers on why this is required:
- before we counted on implicit get to do this test for us... and until https://github.com/Pix4D/concourse-pypi-resource/pull/91 it worked fine... what we called before `pypi-acceptance-test` was testing both put and get steps... after I disabled the implicit get, I basically removed half of the tests...

The approach proposed here adds back the get step tests but in a separate job to conform to OPA rules (no implicit get). This is also clearer and the same approach as in: https://builder.ci.pix4d.com/teams/developers/pipelines/s3-resource-simple-master

**Comment by Pix4D-Janus** [comment]
pipeline_setter_bot: your pipeline is set:
https://builder.ci.pix4d.com/teams/developers/pipelines/concourse-pypi-resource-pipeline-re-add-get-test
	policy check failed:  
 * Violation: Each resource type must configure check_every: 24h. Affected resource types: pypi.test. Documentation: https://pix4dbug.atlassian.net/wiki/x/R4DaKAE 
 * Violation: Some Docker credentials are deprecated. Replaced with 'concourse:((concourse_artifactory_password))'. Found deprecated variables: ((docker_cloud_pix4d_password)), ((docker_cloud_pix4d_user)). Documentation: https://pix4dbug.atlassian.net/wiki/x/FYDQKwE
WARNING: unblocking from the policy check failure for soft enforcement

**Review by aliculPix4D — APPROVED** [review]
Thanks...

File: ci/pipelines/concourse-pypi-resource-pipeline-template.yml (hunk 1 above)
Comment (marco-m-pix4d) [reply ci/pipelines/concourse-pypi-resource-pipeline-template.yml:189]: Can we add a comment somewhere mentioning this rationale?

**Review by marco-m-pix4d — APPROVED** [review]

**Pull request merged**